dist/element.js

```diff
@@ -0,0 +1,14 @@
+function $parcel$interopDefault(a) {
+  return a && a.__esModule ? a.default : a;
+}
```

Comment:
okay, it usually doesn't make sense to include `dist` files in the repo, but since we do that for the images, it doesn't totally make sense to have the `element.js` be in a different dist folder or act completely differently - so for now this is included in the repo. I'm open to suggestions here, but I think this is a small hit to take, especially since the built files aren't using hashed file names, and we only get two (this and the map).